packages/capabilities/src/access.js

```diff
@@ -83,6 +83,11 @@ export const confirm = capability({
   can: 'access/confirm',
   with: DID,
   nb: Schema.struct({
+    /**
+     * Link to the `access/authorize` request that this delegation was created
+     * for.
+     */
+    cause: Schema.link({ version: 1 }),
```

Comment:
should it be optional?

there are many confirms in the db already that may not parse unless its optional?

Reply:
I don't think we should. We don't actually store confirms in DB just send them to user email and they expire in 15 minutes. So practically only way for someone to run into the problem would be if they tried to authorize just before we deploy this and then they click the link after we deployed it.

Given such a low probability and a low impact on not yet officially launched product, I don't think carrying the burden is worth it. If we were in the production I would have had a different opinion.